trillian/ctfe/ct_server/main.go

```diff
@@ -289,12 +290,24 @@ func setupAndRegister(ctx context.Context, client trillian.TrillianLogClient, de
 		glog.Info("Enabling quota for intermediate certificates")
 		opts.CertificateQuotaUser = ctfe.QuotaUserForCert
 	}
+	// Full handler pattern will be of the form "/logs/yyz/ct/v1/add-chain", where "/logs" is the
+	// HandlerPrefix and "yyz" is the c.Prefix for this particular log. Use the default
+	// HandlerPrefix unless the log config overrides it. The custom prefix in
+	// the log configuration intended for use in migration scenarios where logs
+	// have an existing URL path that differs from the global one. For example
+	// if all new logs are served on "/logs/log/..." and a previously existing
+	// log is at "/log/..." this is now supported.
+	lhp := *handlerPrefix
```

Comment:
nit: could we pass in `handlerPrefix` as a parameter so only `main()` touches flags please

Reply:
Done.